app/main.py

```diff
@@ -1,8 +1,53 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
```

Comment:
The constructor parameters are correctly aligned, but make sure to follow the checklist's recommendation to avoid breaking before binary operations.